src/test/java/core/basesyntax/operations/BuyingTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.customExceptions.NoRequiredQuantityOfFruits;
+import core.basesyntax.orderProcessing.OrdersReader;
+import core.basesyntax.orderProcessing.OrdersStorage;
+import core.basesyntax.orderProcessing.StoreService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Map;
+
+public class BuyingTest {
```

Comment:
It seems we need to make one test file. No eto ne tochno :)

Reply:
I thought to divide them because it touches different class/functionality)